x/bank/keeper.go

```diff
@@ -7,10 +7,14 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/x/auth"
 	"github.com/cosmos/cosmos-sdk/x/params"
+	"github.com/tendermint/tendermint/crypto"
 )
 
 var _ Keeper = (*BaseKeeper)(nil)
 
+// Account to send burned coins to
+var BurnedCoinsAccAddr = sdk.AccAddress(crypto.AddressHash([]byte("bankBurnedCoins")))
```

Comment:
why not just set this to `0x00..`?

Reply:
this is really messy, then total supply won't be tracked correctly

Reply:
(I suggest we add a `BurnCoins` function to reduce total supply appropriately)

Reply:
@cwgoes There was a PR (#2939) to do that, but it was decided to be postponed until postlaunch.

Reply:
in the future we can prob have this special address actually burn.  lets do it after launch.